<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WB OZON Data Analitics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>470</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>WB OZON Data Analitics</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>576</width>
      <height>473</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideLeft</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>15</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>Продажи, шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Продажи на площадке, руб.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1 шт.</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>К перечислению после вычета МП, руб.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1 шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Итого комиссия МП</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Реклама МП, руб.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Процент на рекламу</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1 шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>К перечислению после вычета рекламы, руб.</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1 шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Себестоимость 1 шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Доход 1 шт.</string>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Доход 1 шт.</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>true</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>37</red>
        <green>95</green>
        <blue>166</blue>
       </color>
      </brush>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Итого доход, руб.</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Озон</string>
     </property>
     <property name="background">
      <color>
       <red>90</red>
       <green>135</green>
       <blue>236</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BБ</string>
     </property>
     <property name="background">
      <color>
       <red>177</red>
       <green>90</green>
       <blue>243</blue>
      </color>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Итого </string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="9" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="9" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="10" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="10" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="11" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="11" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="12" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="12" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="13" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="13" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="14" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="14" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>519</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Скопировать!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>523</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
